packages/render-fidelity-tools/src/components/renderers/babylon-viewer.ts

```diff
@@ -88,7 +88,10 @@ export class BabylonViewer extends LitElement {
 
     this[$updateSize]();
 
-    const {orbit, target, verticalFoV} = scenario;
+    const {orbit, target, verticalFoV, renderSkybox} = scenario;
+
+    this[$scene].clearColor = new Color4(1, 1, 1, 1);
```

Comment:
At first i put clear color into scenario config, but i have to hard code filament and gltf-pt's clear color anyway, i just make all renderers' clear color hard coded as white